web-app/js/portal/cart/Downloader.js

```diff
@@ -103,21 +103,21 @@ Portal.cart.Downloader = Ext.extend(Ext.util.Observable, {
     _downloadAsynchronously: function(collection, downloadUrl, params) {
         log.debug('downloading asynchronously', downloadUrl);
 
+        this.params = params;
         Ext.Ajax.request({
             url: downloadUrl,
             scope: this,
-            params: params,
             success: this._onAsyncDownloadRequestSuccess,
```

Comment:
You could make the callback function an anonymous function which calls the function we want. ie. `function() { this._onAsyncDownloadRequestSuccess(params) }`. That will work because `params` is in scope for the anonymous function even thought it won't get called until later.